infra/otel/sdk/metric/prometheus/prometheus_test.go

```
@@ -0,0 +1,384 @@
+	addr := r.addr()
+
+	resp, err := http.Get("http://" + addr + metricsPath)
+	require.NoError(t, err)
+	resp.Body.Close()
+	require.Equal(t, http.StatusOK, resp.StatusCode)
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+	require.NoError(t, stop(ctx))
+
+	_, err = http.Get("http://" + addr + metricsPath)
```

golangci-lint (2.12.2)

[error] response body must be closed

(bodyclose)